extensions/ql-vscode/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Add a prompt to the "Quick query" command to encourage users in single-folder workspaces to use "Create query" instead. [#3082](https://github.com/github/vscode-codeql/pull/3082)
 - Remove support for CodeQL CLI versions older than 2.11.6. [#3087](https://github.com/github/vscode-codeql/pull/3087)
 - Preserve focus on results viewer when showing a location in a file. [#3088](https://github.com/github/vscode-codeql/pull/3088)
+- The `dataflowtracking` and `tainttracking` snippets expand to the new module-based interface. [#3091](https://github.com/github/vscode-codeql/pull/3091)
```

Comment:
I'm not completely sure if I got the terminology for snippets correct here 😂

<img width="565" alt="i-have-no-idea" src="https://i.kym-cdn.com/entries/icons/original/000/008/342/ihave.jpg">

Reply:
I don't know either but it looks good enough to me